src/engine/world.rs

```diff
@@ -43,6 +45,8 @@ pub struct World {
     pub input: InputManager,
     /// Asset storage containing meshes, textures, materials, etc.
     pub assets: Arc<AssetStore>,
+    /// Audio scene manager
+    pub audio_scene: AudioScene,
```

Comment:
I'm not sure yet if we wanna have one AudioScene per World but it's definitely gonna be simple to use for now. Let's keep it like this 👍🏻